security/samsung_signature.ksy

```diff
@@ -0,0 +1,36 @@
+meta:
+  id: samsung_signature
+  title: Samsung Crypto Signature for Android Firmware Images
+  license: CC0-1.0
+  encoding: utf-8
+
+doc: |
+  Signature that samsung appends to android images files.
+  Only part of the fields reversed. A phone's bootloader
```

Comment:
_reversed_ = _described_ or _parsed_?

Reply:
`to reverse = to reverse engineer` AFAIK.